### PR TITLE
check if withCredentials has been set on the fileInfo object, if it i…

### PR DESCRIPTION
**Pull request by alogan519**
…s set to false then don't use credentials

### CODE REVIEW

File: src/fileViewer.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ var FileViewer = React.createClass({
 
 		return <IntlFileViewer
 			{...this.props}
+			withCredentials={this.state.info.withCredentials}
```
Comment (alogan519) [inline src/fileViewer.js]: so if withCredentials is undefined it gets handled in alternateViewer and does supply credentials (same as before)

Comment (jstefaniuk-d2l) [inline src/fileViewer.js]: Looks like `this.state.info` has the possibility of being null here if force generic is true

Comment (alogan519) [inline src/fileViewer.js]: Ok, I could mimic the same behaviour mimetype does.

Comment (jstefaniuk-d2l) [inline src/fileViewer.js]: This seems like something pretty specific to even go here, I think the `fileInfo` prop gets passed all the way down to the `alternateViewer`, may be worth just checking for the attribute there

Comment (alogan519) [inline src/fileViewer.js]: Ok, sure that works

**Pull request merged**